### PR TITLE
Bug 24332139 - WEBSERVICE CALL CAN FAIL IF PAYLOAD INCLUDES WSSE:SECU…

### PR DESCRIPTION
**Pull request by monicadragomir**
…RITY HEADER

### CODE REVIEW

**Comment by eclipsewebmaster** [comment]
Issue tracker reference:
https://bugs.eclipse.org/bugs/show_bug.cgi?id=24332139

**Comment by monicadragomir** [comment]
Bug 24332139 - WEBSERVICE CALL CAN FAIL IF PAYLOAD INCLUDES WSSE:SECURITY HEADER

File: jaxws-ri/rt/src/main/java/com/sun/xml/ws/util/xml/XmlUtil.java [hunk 1]
```diff
@@ -87,6 +87,8 @@ public Boolean run() {
             }
     );
 
+    private static boolean RUN_WITH_SECURITY_MANAGER = (System.getSecurityManager() != null);
```
Comment (lukasj) [inline jaxws-ri/rt/src/main/java/com/sun/xml/ws/util/xml/XmlUtil.java]: there may be a chance that SM will be changes during runtime - do we have to care about such case?

File: jaxws-ri/rt/src/test/java/com/sun/xml/ws/util/xml/XmlUtilTest.java [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+        method.setAccessible(true);
+        
+        Field fieldRunWithSecurityManager = com.sun.xml.ws.util.xml.XmlUtil.class.getDeclaredField("RUN_WITH_SECURITY_MANAGER");
+        fieldRunWithSecurityManager.setAccessible(true);
+        Field fieldDisabledBySetting = com.sun.xml.ws.util.xml.XmlUtil.class.getDeclaredField("XML_SECURITY_DISABLED");
+        fieldDisabledBySetting.setAccessible(true);
+        
+        fieldRunWithSecurityManager.set(com.sun.xml.ws.util.xml.XmlUtil.class, true);
+        
+        fieldDisabledBySetting.set(com.sun.xml.ws.util.xml.XmlUtil.class, true);
```
Comment (lukasj) [inline jaxws-ri/rt/src/test/java/com/sun/xml/ws/util/xml/XmlUtilTest.java]: if the test fails here for whatever reason, following tests, if there are any, may be influenced by it as the RUN_WITH_SECURITY_MANAGER field remains set to true. I'd prefer defensive approach here

**Comment by lukasj** [comment]
rebased and going to be merged within #94

**Pull request closed without merging**